spec/controllers/content_items_controller_spec.rb

```diff
@@ -41,6 +41,16 @@
 
       expect(subject).to render_template(:index)
     end
+
+    it "assigns the lists of taxonomies and organisations ordered ASC by title" do
+      organisations = [{ title: "ABC" }, { title: "BCD" }].map { |o| create(:organisation, o) }
+      taxons = [{ title: "ABC" }, { title: "BCD" }].map { |t| create(:taxonomy, t) }
```

Comment:
The title is used to ensure the list returned is in alphabetical order - but I could make the test clearer by creating them in reverse order then testing the title of the first one i.e. it should be `"ABC"`?